platforms/chibios/converters/elite_c_to_rp2040_ce/_pin_defs.h

```diff
@@ -37,3 +37,6 @@
 #define C7 14U
 #define F1 15U
 #define F0 16U
+
+// VBUS detection Pin
+#define USB_VBUS_PIN 19U
```

Comment:
We have tried to avoid non GPIO defines within this file. My current preference would be how it was previously, specified within the `converter.mk` file.

Reply:
Where can i see how it was done previously?

Reply:
Your previous PR

<https://github.com/qmk/qmk_firmware/pull/19470#issuecomment-1369107685>

Reply:
that was a question 😅

Reply:
`OPT_DEFS += -DUSB_VBUS_PIN=19U` so that should work?

Reply:
I cant see the commits now, but i'm certain you also committed those change to the PR. But yes that should work.

Reply:
```┌─[conor@m93p] - [~/Downloads/qmk_firmware] - [2023-01-19 10:17:15]
└─[130] <git:(develop e99b2b6ec8) > qmk compile -kb evyd13/pockettype -km default -e CONVERT_TO=bonsai_c3
Ψ Compiling keymap with make --jobs=1 CONVERT_TO=bonsai_c3 evyd13/pockettype:default


QMK Firmware 0.19.10
Making evyd13/pockettype with keymap default

builddefs/build_keyboard.mk:425: platforms/platforms/platform.mk: No such file or directory
make[1]: *** No rule to make target 'platforms/platforms/platform.mk'.  Stop.
Make finished with errors
make: *** [Makefile:392: evyd13/pockettype:default] Error 1
┌─[conor@m93p] - [~/Downloads/qmk_firmware] - [2023-01-19 10:17:41]
└─[2] <git:(develop e99b2b6ec8) > ```
cant seem to compile with develop

Reply:
@zvecr could you try it, I've pushed the change

Reply:
Seems like I broke it with some recent changes. 19644 should fix it.

Reply:
okay it works